generation/maisi/scripts/inference.py

```diff
@@ -0,0 +1,238 @@
+            "url": "https://drive.google.com/file/d/1jQefG0yJPzSvTG5rIJVHNqDReBTvVmZ0/view?usp=drive_link",
+        },
+        {
+            "path": "models/input_unet3d_data-all_steps1000size512ddpm_random_current_inputx_v1.pt",
+            "url": "https://drive.google.com/file/d/1FtOHBGUF5dLZNHtiuhf5EH448EQGGs-_/view?usp=sharing",
+        },
+        {
+            "path": "models/controlnet-20datasets-e20wl100fold0bc_noi_dia_fsize_current.pt",
+            "url": "https://drive.google.com/file/d/1izr52Whkk56OevNTk2QzI86eJV9TTaLk/view?usp=sharing",
+        },
+        {
+            "path": "models/mask_generation_autoencoder.pt",
+            "url": "https://drive.google.com/file/d/1FzWrpv6ornYUaPiAWGOOxhRx2P9Wnynm/view?usp=drive_link",
+        },
+        {
+            "path": "models/mask_generation_diffusion_unet.pt",
+            "url": "https://drive.google.com/file/d/11SA9RUZ6XmCOJr5v6w6UW1kDzr6hlymw/view?usp=drive_link",
+        },
+        {
+            "path": "configs/candidate_masks_flexible_size_and_spacing_3000.json",
+            "url": "https://drive.google.com/file/d/1yMkH-lrAsn2YUGoTuVKNMpicziUmU-1J/view?usp=sharing",
+        },
+        {
+            "path": "configs/all_anatomy_size_condtions.json",
+            "url": "https://drive.google.com/file/d/1AJyt1DSoUd2x2AOQOgM7IxeSyo4MXNX0/view?usp=sharing",
+        },
+        {
+            "path": "datasets/all_masks_flexible_size_and_spacing_3000.zip",
+            "url": "https://drive.google.com/file/d/16MKsDKkHvDyF2lEir4dzlxwex_GHStUf/view?usp=sharing",
+        },
+    ]
+
+    for file in files:
+        file["path"] = file["path"] if "datasets/" not in file["path"] else os.path.join(root_dir, file["path"])
+        download_url(url=file["url"], filepath=file["path"])
+
+    # ## Read in environment setting, including data directory, model directory, and output directory
+    # The information for data directory, model directory, and output directory are saved in ./configs/environment.json
+    env_dict = json.load(open(args.environment_file, "r"))
+    for k, v in env_dict.items():
+        # Update the path to the downloaded dataset in MONAI_DATA_DIRECTORY
+        val = v if "datasets/" not in v else os.path.join(root_dir, v)
+        setattr(args, k, val)
+        print(f"{k}: {val}")
+    print("Global config variables have been loaded.")
+
+    # ## Read in configuration setting, including network definition, body region and anatomy to generate, etc.
+    #
+    # The information used for both training and inference, like network definition, is stored in "./configs/config_maisi.json". Training and inference should use the same "./configs/config_maisi.json".
+    #
+    # The information for the inference input, like body region and anatomy to generate, is stored in "./configs/config_infer.json". Please feel free to play with it.
+    # - `"num_output_samples"`: int, the number of output image/mask pairs it will generate.
```

Comment:
Perhaps we can remove these comment lines and add a link refer to the README file. Thanks.

Reply:
Done. Thanks